frontend/src/concepts/pipelines/content/PipelineServerActions.tsx

```diff
@@ -68,24 +68,28 @@ const PipelineServerActions: React.FC<PipelineServerActionsProps> = ({ variant,
           <DropdownItem key="view-server-details" onClick={() => setViewOpen(true)}>
             View pipeline server configuration
           </DropdownItem>,
-          <Divider key="separator" />,
-          <DropdownItem
-            onClick={() => {
-              setDeleteOpen(true);
-            }}
-            key="delete-server"
-          >
-            Delete pipeline server
-          </DropdownItem>,
-          ...(variant === 'kebab'
+          ...(pipelinesServer.compatible
             ? [
+                <Divider key="separator" />,
                 <DropdownItem
-                  key="delete"
-                  onClick={() => setDeletePipelinesOpen(true)}
-                  isDisabled={pipelines.length === 0 && versions.length === 0}
+                  onClick={() => {
+                    setDeleteOpen(true);
+                  }}
+                  key="delete-server"
                 >
-                  Delete
+                  Delete pipeline server
                 </DropdownItem>,
+                ...(variant === 'kebab'
+                  ? [
+                      <DropdownItem
+                        key="delete"
+                        onClick={() => setDeletePipelinesOpen(true)}
+                        isDisabled={pipelines.length === 0 && versions.length === 0}
+                      >
+                        Delete
+                      </DropdownItem>,
+                    ]
+                  : []),
```

Comment:
if the server is not compatible we should not show the kebab one either

Reply:
@Gkrumbach07 , I have already taken that into account

![Screenshot 2024-10-30 at 11 19 20 AM](https://github.com/user-attachments/assets/60ddb2dc-b53f-47b6-aacb-9ad85ca1f9d6)